README.md

```diff
@@ -80,82 +82,74 @@ Example output:
+In this section, you will create an Ubuntu 23.10-based TD guest from scratch or convert an existing non-TD guest into one. This can be performed on any Ubuntu 22.04 or newer system and a TDX-specific environment is not required.  
 
-```bash
-  $ wget https://github.com/canonical/tdx/archive/refs/heads/main.zip
-  $ unzip main.zip
-  $ cd tdx-main/
-```
+### Create a New TD Guest Image
 
-- Generate TD guest image
+The base image is an Ubuntu 23.10 cloud image [`ubuntu-23.10-server-cloudimg-amd64.img`](https://cloud-images.ubuntu.com/releases/mantic/release/). You can be customized your preferences by setting these two environment variables before running the script:
 
 ```bash
-  $ (cd  guest-tools/image/ ; sudo ./create-td-image.sh)
+export OFFICIAL_UBUNTU_IMAGE="https://cloud-images.ubuntu.com/releases/mantic/release/"
+export CLOUD_IMG="ubuntu-23.10-server-cloudimg-amd64.img"
 ```
 
-This script will create a TD image based on the Ubuntu 23.10 Cloud Image ubuntu-23.10-server-cloudimg-amd64.img
-and can be found at : https://cloud-images.ubuntu.com/releases/mantic/release/
-
-This base image can be customized by setting two environment variables:
+1. Generate a TD guest image.
 
 ```bash
-  $ export OFFICIAL_UBUNTU_IMAGE="https://cloud-images.ubuntu.com/releases/mantic/release/"
-  $ export CLOUD_IMG="ubuntu-23.10-server-cloudimg-amd64.img"
+cd tdx/guest-tools/image/
+sudo ./create-td-image.sh
 ```
 
-The produced TD guest image will be available at : guest-tools/image/tdx-guest-ubuntu-23.10.qcow2
+The produced TD guest image is `tdx-guest-ubuntu-23.10.qcow2`.
 
-The root password is set to 123456
+The root password is set to `123456`.
 
-### Boot TD Guest with QEMU
+### Convert a Non-TD Guest into a TD Guest
 
-Now that you have a TD guest image, let’s boot it with QEMU.
+If you have an existing Ubuntu 23.10 non-TD guest, you can enable the TDX feature by following these steps.
 
-- Start the TD guest
+1. Boot up your guest.  
 
-If you have downloaded this repository, the script wil be in guest-tools.
+2. Clone this repo.
 
 ```bash
-  $ cd guest-tools
+git clone git@github.com:canonical/tdx.git
 ```
 
-If not, you can download the script as follow:
+3. Run the script.
 
 ```bash
-  $ https://raw.githubusercontent.com/canonical/tdx/dev-review/guest-tools/run_td.sh
-  $ chmod a+x run_td.sh
+cd tdx
+sudo ./setup-tdx-guest.sh
 ```
+4. Shutdown the guest.  
 
-To run the TD:
+## Boot TD Guest
+
+1. Now that you have a TD guest image, let’s boot it with the provided script.
 
 ```bash
-  $ TD_IMG=<path_to_td_qcow2> ./run_td.sh
+cd tdx/guest-tools
+TD_IMG=<path_to_td_qcow2_image> ./run_td.sh
 ```
 
-- Log into the guest
+2. Log into the guest. 
 
-The script will run the TD in the background and if the guest image has been created from scratch previously,
-it can be accessed via SSH:
+NOTE: The default credentials for a TD guest created from scratch is `root/123456`.   
 
 ```bash
-  $ ssh -p 10022 root@localhost
+ssh -p 10022 root@localhost
```

Comment:
This command to access the TD is solely valid for TD created from scratch (ssh is configured to accept ssh login under root, ..), if the user converts an existing VM into TD, access to the TD will be user specific, this is the reason why i put this boot section just after the "Create TD image from scratch", i think we can keep your proposed structure, it makes sense but we might want to point out this detail

Reply:
Here's what I changed it to:
      NOTE: The example below uses the credentials for a TD guest created from scratch.    
      If you converted your own guest, please use your original credentials.

Reply:
it is not really a big deal here but to be precise, I meant, besides the credentials, the ssh command also is specific to TD created from scratch, more precisely, the proposed ssh user root is only applicable to the TD created from scratch, if the user uses an existing VM, he/she might want to access to the TD under a different username ...

Reply:
When I use "credentials", I am referring to both username and password.  Is that ok?

Reply:
ok, got it